it-tests/settings/NodeLabelSettings.test.ts

```diff
@@ -14,8 +14,8 @@ describe('User Settings', function () {
 
     const locators = {
         TimerComponent: {
-            timer: By.xpath(`//\*[name()='g' and starts-with(@data-id,'timer')]`),
-            label: By.xpath(`//\*[name()='g' and starts-with(@class,'pf-topology__node__label')]`)
+            timer: `g[data-id^='timer'][data-kind='node']`,
+            label: `.custom-node__label`,
```

Comment:
This selector changed because there's no more topology label but rather a standard text.